tools/Cron_jobs/quarantine_maint.php

```diff
@@ -84,6 +84,11 @@ function quarantine_clean()
                         // Update the quarantine flag
                         $sql = "UPDATE maillog SET timestamp=timestamp, quarantined = NULL WHERE id='$id'";
                         dbquery($sql);
+                        //ASU MODIFIED CODE BEGIN
+                        //remove any corresponding record in mod_release
+                        $sql = "DELETE FROM mod_release WHERE msg_id = '$id'";
```

Comment:
Add a flag in conf.PHP and  an if nere to bypass this query if autorelease is not enabled